src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const longTermRent = 7;
+  const discountLongTerm = 50;
+  const shortTermRent = 3;
+  const discountShortTerm = 20;
+  const oneDayCost = 40;
```

Comment:
those should be outside of and uppercased - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2